### ISSUE TITLE
bug related to item_number?

### ISSUE THREAD

**Issue by dnjackson** [maripuncel/mmiyachi_proj2#5]
From https://github.com/6170/mmiyachi_proj2/blob/master/app/controllers/carts_controller.rb:

   @item = Item.find_by_id(@cart_item.item_number)

Are you confusing item_number meaning "count" with an item_number that might have been used to identify the item?

**Comment by maripuncel** [comment]
No, the item_numer of a cart item is the item id of the normal item it is mirroring
